src/graphql/sampling.rs

```diff
@@ -163,6 +163,10 @@ impl SamplingPolicy {
         self.inner.dst_ip.as_ref().map(ToString::to_string)
     }
 
+    async fn node(&self) -> Option<String> {
+        self.inner.node.as_ref().map(ToString::to_string)
```

Comment:
Could you explain why you didn't use `clone`?

Reply:
Thank you for the comment. I changed it to clone, which I agree as simpler and straightforward.